src/meta/play-meta.js

```diff
@@ -58,6 +59,17 @@ const plays = [
     level: 'Intermediate',
     tags: 'Recursion, Tree',
     github: 'green-roots'
+  }, {
+    id: 'pl-counter', 
+    name: 'Counter', 
+    description: 'A simple counter',
```

Comment:
Can you please provide some more details as to what does the counter supposed to do?

Reply:
It's a simple counter demonstrating the use of `state` and `props` in React! It will reset after the specified value is reached.